modules/veteran_verification/spec/models/disability_rating_spec_v1.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
```

Comment:
I don't think this spec is running with the others. Possibly because the filename ends with _spec_v1.rb instead of just _spec.rb. When I run it by itself it is failing.

Reply:
Hmm, I thought I had run that rspec locally, but then when through quite a bit of Rubocop refactoring that was being enforced on existing code due to the v0/v1 folder addition.

I have corrected this and should be passing now.